### ISSUE TITLE
FNX-5260 ⁃ [Bug] A11y - Unlabeled button:  security level indicator

### ISSUE THREAD

**Issue by sv-ohorvath** [mozilla-mobile/fenix#1815]
## Steps to reproduce
1. Use talkback to navigate to the security level indicator icon in the url bar.
    1. Expected behavior
The button should have a description.
    1. Actual behavior
Talkback reads "Unlabeled button"
    1. Device information

* Android device: HTC 10 (Android 8.0)
* Fenix version: 4/19 build

**Comment by ekager** [comment]
Just a note - based on some mocks I've seen, we may end up changing what displays in the URL bar at this location.

**Comment by asadotzler** [comment]
Same as issue #1392 ?

**Comment by asadotzler** [comment]
This is definitely a duplicate. Please close this issue. Thanks.
